### PR TITLE
Iambmelt/caitbal nitpicks

### PR DESCRIPTION
**Pull request by iambmelt**
Relates to [#79](https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/pull/79)

### CODE REVIEW

File: graphsdk/src/main/java/com/microsoft/graph/generated/BaseDirectoryObjectReferenceRequest.java [hunk 1]
```diff
@@ -38,7 +38,7 @@ public void delete(final ICallback<DirectoryObject> callback) {
     }
 
     public DirectoryObject delete() throws ClientException {
-       return send(HttpMethod.DELETE, null);
+        return send(HttpMethod.DELETE, null);
```
Comment (iambmelt) [inline graphsdk/src/main/java/com/microsoft/graph/generated/BaseDirectoryObjectReferenceRequest.java:41]: Appears to be an artifact of an existing formatting error in the template. I don't stress this stuff, but it's there.

File: graphsdk/src/main/java/com/microsoft/graph/generated/BaseDirectoryObjectReferenceRequest.java [hunk 2]
```diff
@@ -49,7 +49,7 @@ public DirectoryObject delete() throws ClientException {
      */
     public IDirectoryObjectReferenceRequest select(final String value) {
         getQueryOptions().add(new QueryOption("$select", value));
-        return (DirectoryObjectReferenceRequest)this;
+        return (DirectoryObjectReferenceRequest) this;
     }
```
Comment (iambmelt) [inline graphsdk/src/main/java/com/microsoft/graph/generated/BaseDirectoryObjectReferenceRequest.java:53]: More template stuff

File: graphsdk/src/main/java/com/microsoft/graph/generated/BaseDirectoryObjectReferenceRequest.java [hunk 3]
```diff
@@ -60,26 +60,27 @@ public IDirectoryObjectReferenceRequest select(final String value) {
      */
     public IDirectoryObjectReferenceRequest expand(final String value) {
         getQueryOptions().add(new QueryOption("$expand", value));
-        return (DirectoryObjectReferenceRequest)this;
+        return (DirectoryObjectReferenceRequest) this;
```
Comment (iambmelt) [inline graphsdk/src/main/java/com/microsoft/graph/generated/BaseDirectoryObjectReferenceRequest.java:63]: Here too

File: graphsdk/src/main/java/com/microsoft/graph/generated/BaseUserReferenceRequest.java [hunk 4]
```diff
@@ -38,7 +38,7 @@ public void delete(final ICallback<User> callback) {
     }
 
     public User delete() throws ClientException {
-       return send(HttpMethod.DELETE, null);
+        return send(HttpMethod.DELETE, null);
```
Comment (iambmelt) [inline graphsdk/src/main/java/com/microsoft/graph/generated/BaseUserReferenceRequest.java:41]: Same here - template stuff

File: graphsdk/src/main/java/com/microsoft/graph/generated/BaseUserReferenceRequest.java [hunk 5]
```diff
@@ -49,7 +49,7 @@ public User delete() throws ClientException {
      */
     public IUserReferenceRequest select(final String value) {
         getQueryOptions().add(new QueryOption("$select", value));
-        return (UserReferenceRequest)this;
+        return (UserReferenceRequest) this;
```
Comment (iambmelt) [inline graphsdk/src/main/java/com/microsoft/graph/generated/BaseUserReferenceRequest.java:52]: More

File: graphsdk/src/main/java/com/microsoft/graph/generated/BaseUserReferenceRequest.java [hunk 6]
```diff
@@ -60,26 +60,27 @@ public IUserReferenceRequest select(final String value) {
      */
     public IUserReferenceRequest expand(final String value) {
         getQueryOptions().add(new QueryOption("$expand", value));
-        return (UserReferenceRequest)this;
+        return (UserReferenceRequest) this;
```
Comment (iambmelt) [inline graphsdk/src/main/java/com/microsoft/graph/generated/BaseUserReferenceRequest.java:63]: More

**Comment by iambmelt** [comment]
Related: #39

**Review by caitlinrussell — APPROVED** [review]
Looks good

**Pull request merged**